exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -605,4 +605,5 @@ drill.exec.options: {
     store.kafka.poll.timeout: 200,
     web.logs.max_lines: 10000,
     window.enable: true,
+    storage.list_files_recursively: true
```

Comment:
I would suggest we keep the default false for backward compatibility - if not, show files command output may explode for very high level directories, and may be less helpful for exploratory purposes.

Reply:
ah...Is this setting only applicable when querying the INFORMATION_SCHEMA.`FILES` table? And SHOW FILES command will continue to work as it does today? If so, sorry - please ignore my previous comment.

Reply:
Sounds fair, made the changes. Thanks!